### ISSUE TITLE
Move Storybook to a branch

### ISSUE THREAD

**Issue by C-STYR** [gravitational/next#814]
- Remove Storybook and all storybook entries from `main` and place them into a branch until further notice. 
- Update scripts accordingly

**Comment by C-STYR** [comment]
@iAdramelk this didn't come from me. 
Chromatic is another story, and this may not be a permanent move, but for now proceed please.

**Comment by alexwolfe** [comment]
Lets put this in a branch for now. Currently its overhead and its not being used for any effective testing at the moment. It can be merged back in when it is beneficial to the project. Right now its simply overhead and providing not major benefit. If you can create testing on this branch and make a case for it to be merged back into main then great. But updating storybook is not the priority for us right now. There are other tasks that have been already assigned and should be the focus at the moment.

**Comment by iAdramelk** [comment]
@alexwolfe I strongly disagree here. We plan to split repo into three parts and from that I understand we also plan to reduce number of components used on site. If we remove Stortybook now, it is almost 100% that we would not be able to merge it back later after these changes. 

Right now building Storybook in disabled at Vercel builds so it does not affect build speed or reliability in any way, and it also does not affect development process in any way. If we don't want to experiment with Cromatic now, let's postpone it for later, but lets not remove Storybook until we discussed our testing process.

**Comment by alexwolfe** [comment]
We'll hold off until we've made some decisions regarding splitting into different branches and a test process.
